bcda/alr/gen/synthetic.go

```diff
@@ -66,6 +66,12 @@ var valuegen map[*regexp.Regexp]func() string = map[*regexp.Regexp]func() string
 			return strconv.FormatFloat(res, 'f', 3, 64)
 		})
 	},
+	regexp.MustCompile("_EXCLUDED$"): func() string {
```

Comment:
Can we change this to "EXCLUDED$" to capture this too:
<img width="267" alt="Screen Shot 2021-03-03 at 6 59 45 PM" src="https://user-images.githubusercontent.com/14236737/109889065-a73eb800-7c52-11eb-9568-98e7d7eac9e5.png">